docs/_posts/examples/3400-01-30-3d-buildings.html

```diff
@@ -14,11 +14,17 @@
     zoom: 15,
     pitch: 45,
     bearing: -17.6,
+    hash: true,
     container: 'map'
 });
 
-// the 'building' layer in the mapbox-streets vector source contains building-height
-// data from OpenStreetMap.
+// The 'building' layer in the mapbox-streets vector source contains building-height
+// data from OpenStreetMap. Insert the layer beneath any symbol layer.
+var layers = map.getStyle().layers.reverse();
```

Comment:
This call needs to be inside the `load` handler. Fixed in 6d6c5d92d8e89e8d6c4e608cddaf1febe43c47b1.